docs/quickstart.md

```diff
@@ -265,7 +267,7 @@ This should give you a response like the following:
     {
       "name": "predict",
       "datatype": "FP32",
```

Comment:
One thing I noticed when running the quickstart REST example here is that the return datatype is `INT64`, similar to the gRPC example before it.

```
MODEL_NAME=example-sklearn-isvc
curl -X POST -k http://localhost:8008/v2/models/${MODEL_NAME}/infer -d '{"inputs": [{ "name": "predict", "shape": [1, 64], "datatype": "FP32", "data": [0.0, 0.0, 1.0, 11.0, 14.0, 15.0, 3.0, 0.0, 0.0, 1.0, 13.0, 16.0, 12.0, 16.0, 8.0, 0.0, 0.0, 8.0, 16.0, 4.0, 6.0, 16.0, 5.0, 0.0, 0.0, 5.0, 15.0, 11.0, 13.0, 14.0, 0.0, 0.0, 0.0, 0.0, 2.0, 12.0, 16.0, 13.0, 0.0, 0.0, 0.0, 0.0, 0.0, 13.0, 16.0, 16.0, 6.0, 0.0, 0.0, 0.0, 0.0, 16.0, 16.0, 16.0, 7.0, 0.0, 0.0, 0.0, 0.0, 11.0, 13.0, 12.0, 1.0, 0.0]}]}'
```
```
{"model_name":"example-sklearn-isvc__isvc-6b2eb0b8bf","outputs":[{"name":"predict","datatype":"INT64","shape":[1,1],"data":[8]}]}
```

Reply:
oh, ya, I missed that one. thx